core/src/main/groovy/noe/ews/utils/AbstractJwsNameHelper.groovy

```diff
@@ -0,0 +1,128 @@
+package noe.ews.utils
+
+import groovy.util.logging.Slf4j
+import noe.common.utils.Platform
+import noe.common.utils.Version
+
+/**
+ * JWS Name helper provides logic behind JWS ZIP filenames. These filenames differ based on the EWS version, as well as
+ * old vs new naming. If you require old naming, toggle compatibility mode. Otherwise, new naming will be provided.
+ * Old naming example: jws-application-server-5.0.0-RHEL7-x86_64.zip
```

Comment:
Maybe add the new natives naming here as well?

Reply:
I think here it can be skipped